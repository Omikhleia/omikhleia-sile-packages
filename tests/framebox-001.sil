\begin{document}
\script[src=packages/framebox]
\nofolios

A \framebox{square box.}

A \framebox[shadow=true]{square box with dropped shadow.}

A \framebox[shadow=true, bordercolor=#b94051, fillcolor=#ecb0b8, shadowcolor=220]{colored square box.}

A \roundbox{plain rounded box.}

A \roundbox[shadow=true]{rounded box with dropped shadow.}

A \roundbox[shadow=true, bordercolor=#b94051, fillcolor=#ecb0b8, shadowcolor=220]{fancy colored rounded box.}

\end{document}