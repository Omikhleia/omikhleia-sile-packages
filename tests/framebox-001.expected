Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	30.5447
Set font 	Gentium Plus;10;400;;normal;;LTR
T	36 w=5.9521	(A)
Draw SVG	30.5447	51.5727	13.9121	1.0000	0 4.34375 51.57267 13.91211 re 0 0 0 RG 1 1 1 rg 0.4 w B
Mx 	25.4995
T	86 84 88 68 85 72 w=27.4805	(square)
Mx 	55.6454
T	69 82 91 w=15.1367	(box)
Mx 	70.7821
T	17 w=2.2900	(.)
Mx 	14.8819
My 	45.4568
T	36 w=5.9521	(A)
Draw SVG	45.4568	144.1217	13.9121	1.0000	3 7.34375 144.12166 13.91211 re 0 0 0 rg f 0 4.34375 144.12166 13.91211 re 0 0 0 RG 1 1 1 rg 0.4 w B
Mx 	25.4890
T	86 84 88 68 85 72 w=27.4805	(square)
Mx 	55.6244
T	69 82 91 w=15.1367	(box)
Mx 	73.4160
T	90 76 87 75 w=18.7305	(with)
Mx 	94.8014
T	71 85 82 83 83 72 71 w=34.6045	(dropped)
Mx 	132.0608
T	86 75 68 71 82 90 w=31.2598	(shadow)
Mx 	163.3206
T	17 w=2.2900	(.)
Mx 	14.8819
My 	63.3690
T	36 w=5.9521	(A)
Draw SVG	63.3690	85.1480	13.9121	1.0000	3 7.34375 85.14803 13.91211 re 0.86275 g f 0 4.34375 85.14803 13.91211 re 0.72549 0.25098 0.31765 RG 0.92549 0.6902 0.72157 rg 0.4 w B
Mx 	25.4955
T	70 82 79 82 85 72 71 w=30.9180	(colored)
Mx 	59.0748
T	86 84 88 68 85 72 w=27.4805	(square)
Mx 	89.2167
T	69 82 91 w=15.1367	(box)
Mx 	104.3535
T	17 w=2.2900	(.)
Mx 	14.8819
My 	81.2811
T	36 w=5.9521	(A)
Draw SVG	81.2811	82.4003	13.9121	1.0000	5 4.34375 m 77.40028 4.34375 l 80.1617 4.34375 82.40028 6.58233 82.40028 9.34375 c 82.40028 13.25586 l 82.40028 16.01728 79.63885 18.25586 77.40028 18.25586 c 5 18.25586 l 2.23858 18.25586 0 15.49444 0 13.25586 c 0 9.34375 l 0 6.58233 2.76142 4.34375 5 4.34375 c 0 0 0 RG 1 1 1 rg 0.4 w B
Mx 	25.4961
T	83 79 68 76 81 w=20.8252	(plain)
Mx 	48.9833
T	85 82 88 81 71 72 71 w=34.8242	(rounded)
Mx 	86.4696
T	69 82 91 w=15.1367	(box)
Mx 	101.6063
T	17 w=2.2900	(.)
Mx 	14.8819
My 	96.1932
T	36 w=5.9521	(A)
Draw SVG	96.1932	151.4622	13.9121	1.0000	8 7.34375 m 149.46218 7.34375 l 152.2236 7.34375 154.46218 9.58233 154.46218 12.34375 c 154.46218 16.25586 l 154.46218 19.01728 151.70075 21.25586 149.46218 21.25586 c 8 21.25586 l 5.23858 21.25586 3 18.49444 3 16.25586 c 3 12.34375 l 3 9.58233 5.76142 7.34375 8 7.34375 c 0 0 0 rg f 5 4.34375 m 146.46218 4.34375 l 149.2236 4.34375 151.46218 6.58233 151.46218 9.34375 c 151.46218 13.25586 l 151.46218 16.01728 148.70075 18.25586 146.46218 18.25586 c 5 18.25586 l 2.23858 18.25586 0 15.49444 0 13.25586 c 0 9.34375 l 0 6.58233 2.76142 4.34375 5 4.34375 c 0 0 0 RG 1 1 1 rg 0.4 w B
Mx 	25.4882
T	85 82 88 81 71 72 71 w=34.8242	(rounded)
Mx 	62.9665
T	69 82 91 w=15.1367	(box)
Mx 	80.7573
T	90 76 87 75 w=18.7305	(with)
Mx 	102.1419
T	71 85 82 83 83 72 71 w=34.6045	(dropped)
Mx 	139.4005
T	86 75 68 71 82 90 w=31.2598	(shadow)
Mx 	170.6603
T	17 w=2.2900	(.)
Mx 	14.8819
My 	114.1053
T	36 w=5.9521	(A)
Draw SVG	114.1053	117.6620	14.0098	1.0000	8 7.44141 m 115.66199 7.44141 l 118.42342 7.44141 120.66199 9.67998 120.66199 12.44141 c 120.66199 16.45117 l 120.66199 19.2126 117.90057 21.45117 115.66199 21.45117 c 8 21.45117 l 5.23858 21.45117 3 18.68975 3 16.45117 c 3 12.44141 l 3 9.67998 5.76142 7.44141 8 7.44141 c 0.86275 g f 5 4.44141 m 112.66199 4.44141 l 115.42342 4.44141 117.66199 6.67998 117.66199 9.44141 c 117.66199 13.45117 l 117.66199 16.2126 114.90057 18.45117 112.66199 18.45117 c 5 18.45117 l 2.23858 18.45117 0 15.68975 0 13.45117 c 0 9.44141 l 0 6.67998 2.76142 4.44141 5 4.44141 c 0.72549 0.25098 0.31765 RG 0.92549 0.6902 0.72157 rg 0.4 w B
Mx 	25.4919
T	73 68 81 70 92 w=22.5195	(fancy)
Mx 	50.6692
T	70 82 79 82 85 72 71 w=30.9180	(colored)
Mx 	84.2451
T	85 82 88 81 71 72 71 w=34.8242	(rounded)
Mx 	121.7271
T	69 82 91 w=15.1367	(box)
Mx 	136.8638
T	17 w=2.2900	(.)
End page
Finish
